### ISSUE TITLE
The cc runtime doesn't support the 'devices' OCI config section

### ISSUE THREAD

**Issue by dlespiau** [01org/cc-oci-runtime#137]
Docker asks us to create the /dev/fuse file by default, we should find a way do to that indeed - as it's there by default, it's probably because it was a widely requested feature? It needs:
- support in the VM kernel
- someone to create that device and/or bind mount into the workload /dev

From docker's OCI config.json file:

```
"devices":[{"path":"/dev/fuse","type":"c","major":10,"minor":229,"fileMode":438,"uid":0,"gid":0}]
```

**Comment by jodh-intel** [comment]
See: https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#devices

**Comment by devimc** [comment]
Will be fixed in 3.0 - https://github.com/clearcontainers/runtime
